integrations/integrations.json

```diff
@@ -19557,6 +19557,116 @@
+                    }
+                },
+                "info_provided_to_referring_integrations": {
+                    "description": ""
+                },
+                "keywords": [
+                    "process counts",
+                    "threads",
+                    "context switch"
+                ],
+                "most_popular": false
+            },
+            "overview": "# System thermal zone\n\nPlugin: windows.plugin\nModule: PerflibThermalZone\n\n## Overview\n\nPerflib provides different statistical methods about Microsoft Windows environment. This collector query for\nObject 'Thermal Zone Information' to show actual temperature per zone.\n\n\n\n\nThis collector is only supported on the following platforms:\n\n- windows\n\nThis collector only supports collecting metrics from a single instance of this integration.\n\n\n### Default Behavior\n\n#### Auto-Detection\n\nThe collector auto-detects all metrics. No configuration is needed.\n\n\n#### Limits\n\nThe default configuration for this integration does not impose any limits on data collection.\n\n#### Performance Impact\n\nThe default configuration for this integration is not expected to impose a significant performance impact on the system.\n",
+            "setup": "## Setup\n\n### Prerequisites\n\nNo action required.\n\n### Configuration\n\n#### File\n\nThe configuration file name for this integration is `netdata.conf`.\nConfiguration for this specific integration is located in the `plugin:windows` section within that file.\n\nThe file format is a modified INI syntax. The general structure is:\n\n```ini\n[section1]\n    option1 = some value\n    option2 = some other value\n\n[section2]\n    option3 = some third value\n```\nYou can edit the configuration file using the `edit-config` script from the\nNetdata [config directory](/docs/netdata-agent/configuration/README.md#the-netdata-config-directory).\n\n```bash\ncd /etc/netdata 2>/dev/null || cd /opt/netdata/etc/netdata\nsudo ./edit-config netdata.conf\n```\n#### Options\n\n\n\nThere are no configuration options.\n\n#### Examples\nThere are no configuration examples.\n\n",
+            "troubleshooting": "",
+            "alerts": "## Alerts\n\nThere are no alerts configured by default for this integration.\n",
+            "metrics": "## Metrics\n\nMetrics grouped by *scope*.\n\nThe scope defines the instance that the metric belongs to. An instance is uniquely identified by a set of labels.\n\n\n\n### Per System thermal zone instance\n\n\n\nThis scope has no labels.\n\nMetrics:\n\n| Metric | Dimensions | Unit |\n|:------|:----------|:----|\n| system.thermalzone_temperature | temperature | Celsius |\n\n",
+            "integration_type": "collector",
+            "id": "windows.plugin-PerflibThermalZone-System_thermal_zone",
+            "edit_link": "https://github.com/netdata/netdata/blob/master/src/collectors/windows.plugin/metadata.yaml",
+            "related_resources": ""
+        },
```

Comment:
```suggestion
```

Same as above, don't include such changes, they are meant to be for previewing on your side that the meta is valid.